egs/chime5/s5/local/score_for_submit.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+# Copyright 2012-2014  Johns Hopkins University (Author: Daniel Povey, Yenda Trmal)
+# Apache 2.0
+
```

Comment:
you could add explanation what the script does (probably also to the help message)